### PR TITLE
Add SignInGateVII test and variant design

### PR DESCRIPTION
**Pull request by vlbee**
## What does this change?

Add SignInGateVII test and variant design

image

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by coldlink — APPROVED** [review]
Looking good to me!

I take it when #22618 is merged in, this PR will need updating to add the ad tracking?

**Comment by vlbee** [comment]
yes, this will need to take on the ad tracking. Waiting to hear back if they want a gate=dismissed on that. Will chase today.

**Comment by vlbee** [comment]
Revised overlay UI
image
image

Mobile: 
image

**Comment by vlbee** [comment]
A privacy setting link to the CMP popup module has now been added. 

@GHaberis @ripecosta - I had to make a slight tweak to the [`common/modules/ui/cmp-ui.js`](https://github.com/guardian/frontend/pull/22644/files?file-filters%5B%5D=.js&file-filters%5B%5D=.scss#diff-8900caf760769ee89c2925e54587629bR8) file in order to show the CMP module from the sign in  gate.  Please let me know if this is a problem or if there is an alternative way I should integrate it?

cmp

**Review by ripecosta — APPROVED** [review]
All good as far as CMP UI goes 👍 
Happy to see it being used by other teams too!

File: static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/design/gate-test-vii-variant.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            <h1 class="signin-gate__header--text--var">Register for free and <br>continue reading</h1>
+        </div>
+        <div class="signin-gate__benefits--var signin-gate__margin-top--var">
+            <p class="signin-gate__benefits--text--var">
+                It’s important to say this is not a step towards a paywall
+            </p>
+        </div>
+        <div class="signin-gate__paragraph--var">
+            <p class="signin-gate__paragraph--text--var">
+               We need more readers to register with us to help sustain our independent, quality journalism. Without you taking this simple step, we miss out on revenues from personalised advertising - a critical source of funding for our future.
```
Comment (paperboyo) [inline static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/design/gate-test-vii-variant.js:34]:
```suggestion
               We need more readers to register with us to help sustain our independent, quality journalism. Without you taking this simple step, we miss out on revenues from personalised advertising – a critical source of funding for our future.
```

File: static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/design/gate-test-vii-variant.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            </p>
+            <p class="signin-gate__paragraph--text--var">
+               Through doing so, you'll help ensure that our reporting remains freely available to everyone, and if we recognise you when you come back, we can improve your news experience too.
```
Comment (paperboyo) [inline static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/design/gate-test-vii-variant.js:37]:
```suggestion
               Through doing so, you’ll help ensure that our reporting remains freely available to everyone, and if we recognise you when you come back, we can improve your news experience too.
```

File: static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/design/gate-test-vii-variant.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+               You can still control your own <a id="js-signin-gate__privacy" class="signin-gate__link--var">privacy settings</a>. Thank you.
+            </p>
+        </div>
+        <div class="signin-gate__buttons">
+            <a class="signin-gate__button signin-gate__button--primary--var js-signin-gate__register-button" href="${signInUrl}">
+                Register for free
+            </a>
+            <a class="signin-gate__dismiss--var js-signin-gate__dismiss" href="#maincontent">I'll do it later</a>
```
Comment (paperboyo) [inline static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/design/gate-test-vii-variant.js:45]:
```suggestion
            <a class="signin-gate__dismiss--var js-signin-gate__dismiss" href="#maincontent">I’ll do it later</a>
```

**Comment by paperboyo** [comment]
Just some minor typographic suggestions. Thx! 👋

**Pull request merged**